ComfortZoneApp/app/src/main/java/com/example/comfortzone/Fragments/FlightFragment.java

```diff
@@ -53,8 +54,8 @@ public void maybeUpdateCitiesList() {
         AllWeathersDatabase db = AllWeathersDatabase.getDbInstance(getContext().getApplicationContext());
         Long timeNow = System.currentTimeMillis();
         Long hourAgo = timeNow - TimeUnit.HOURS.toMillis(1L);
-        List<WeatherData> timesGreater = db.weatherDao().getUploadTimes(hourAgo);
-        if (timesGreater.isEmpty()) {
+        List<WeatherData> timesBeforeHour = db.weatherDao().getUploadTimes(hourAgo);
+        if (!timesBeforeHour.isEmpty()) {
```

Comment:
i guess this also works (referring to my previous comment) - we are clearing only if we have local data before hour ago